src/transformers/models/owlvit/modeling_owlvit.py

```diff
@@ -462,7 +524,7 @@ class OwlViTEncoderLayer(nn.Module):
     def __init__(self, config: OwlViTConfig):
```

Comment:
```suggestion
    # Ignore Copy
    def __init__(self, config: OwlViTConfig):
```
This should fix the failing quality tests